shortener/urls/forms.py

```diff
@@ -0,0 +1,18 @@
+from django import forms
+from .models import Url
+from django.forms.widgets import TextInput
+
+
+class UrlForm(forms.ModelForm):
+
+    long_url = forms.URLField(
+        max_length=200, 
+        label="URL",
+        #  help_text="Please enter the URL of the page.", 
+        initial="http://",
```

Comment:
Super helpful for making the app more user friendly and reducing your own workload.  Nice one!